### PR TITLE
Introduce view display option view weights.

### PR DESCRIPTION
**Pull request by stemount**
To prevent loads of notices.

### CODE REVIEW

File: web/modules/custom/par_data/src/Entity/ParDataAdvice.php [hunk 1]
```diff
@@ -117,6 +118,7 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDisplayOptions('view', [
         'label' => 'hidden',
         'type' => 'text_default',
+        'weight' => 0
```
Comment (jon-edery) [inline web/modules/custom/par_data/src/Entity/ParDataAdvice.php]: FAIL

File: web/modules/custom/par_data/src/Entity/ParDataAuthority.php [hunk 2]
```diff
@@ -201,6 +204,7 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDisplayOptions('view', [
         'label' => 'hidden',
         'type' => 'text_default',
+        'weight' => 0
```
Comment (jon-edery) [inline web/modules/custom/par_data/src/Entity/ParDataAuthority.php]: Fail

File: web/modules/custom/par_data/src/Entity/ParDataEnforcementAction.php [hunk 3]
```diff
@@ -241,6 +242,7 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDisplayOptions('view', [
         'label' => 'hidden',
         'type' => 'text_default',
+        'weight' => 0
```
Comment (jon-edery) [inline web/modules/custom/par_data/src/Entity/ParDataEnforcementAction.php]: ","

File: web/modules/custom/par_data/src/Entity/ParDataEnforcementNotice.php [hunk 4]
```diff
@@ -177,6 +180,7 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDisplayOptions('view', [
         'label' => 'hidden',
         'type' => 'text_default',
+        'weight' => 0
```
Comment (jon-edery) [inline web/modules/custom/par_data/src/Entity/ParDataEnforcementNotice.php]: ","

**Comment by stemount** [comment]
@jon-edery thanks,

**Pull request merged**